_spinnaker/configure_ingress.md

```diff
@@ -1,162 +1,135 @@
 
-It's useful to test the names work at this point.  Point a browser to
-`gate.demo.armory.io/health` and verify you get a JSON response, and try
-`demo.armory.io` (substituting your own hostnames, of course) and verify
-you get at least some of the UI to load (it may not load completely because
-we haven't finished configuring the hostnames).
+```
+  export NAMESPACE={namespace}
+ 
+  kubectl expose service -n ${NAMESPACE} spin-gate --type LoadBalancer \
+  --port 8084 \
+  --target-port 8084 \
+  --name spin-gate-public
+ 
+  kubectl expose service -n ${NAMESPACE} spin-deck --type LoadBalancer \
+  --port 9000 \
+  --target-port 9000 \
+  --name spin-deck-public
+```
 
-### Update the Internal URLs in Spinnaker
+Once these Services have been created, we’ll need to update our Spinnaker deployment so that the UI understands where the API is located. To do this, we’ll use Halyard to override the base URL for both the API and the UI and then redeploy Spinnaker.
 
-The reason your UI may not have been working completely when you tried it
-in the previous step is because Spinnaker is expecting to find Gate served
-off `localhost`.  We need to configure it to use the new hostnames:
+```
+  export NAMESPACE={namespace}
+  export API_URL=$(kubectl get svc -n $NAMESPACE spin-gate-public -o jsonpath='{.status.loadBalancer.ingress[0].hostname}')
+  export UI_URL=$(kubectl get svc -n $NAMESPACE spin-deck-public -o jsonpath='{.status.loadBalancer.ingress[0].hostname}')
+  hal config security api edit --override-base-url http://${API_URL}:8084
+  hal config security ui edit --override-base-url http://${UI_URL}:9000
+  hal deploy apply
+```
 
+### Cleanup
+If you’d like to undo what we’ve done in the steps above you can simply delete the Kubernetes Services and reset the UI and API endpoints.
 ```
-$ hal config security api edit --override-base-url http://gate.demo.armory.io
-$ hal config security ui edit --override-base-url http://demo.armory.io
+$ export NAMESPACE={namespace}
+$ kubectl delete svc -n ${NAMESPACE} spin-gate-public spin-deck-public
+$ hal config security api edit --override-base-url http://localhost:8084
+$ hal config security ui edit --override-base-url http://localhost:9000
 $ hal deploy apply
 ```
 
-Once this change has settled, try hitting the Deck URL again, and verify
-everything is working.  You can now use your DNS names to access your
-Spinnaker instance.
-
-## Secure with SSL
-
-This tutorial presumes you've already created a certificate in the AWS
-Certificate Manager.
-
-### Edit Ingress Manifest
+### Secure with SSL
 
-To enable SSL, edit the existing Load Balancer (either in Spinnaker, or using
-`kubectl -n demo edit ingress demo-ingress`) to replace this line:
+This tutorial presumes you've already created a certificate in the AWS Certificate Manager.
 
+* Edit the LoadBalancer service `spin-gate-public` and  `spin-deck-public` we will include 3 annotations for each.
 ```
```

Comment:
I think you need a blank line before the \``` in order to make this work -- this also throws the rest of the page "off" because the closing \``` is seen as an opening set.